en/concept/system_startup.md

```diff
@@ -68,34 +68,49 @@ The resulting boot log has detailed information about the boot sequence and shou
 
 #### Common boot failure causes
 
-  * For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
-  * A software fault or assertion resulting in a stack trace
+- For custom applications: The system was out of RAM. Run the `free` command to see the amount of free RAM.
+- A software fault or assertion resulting in a stack trace
 
 ### Replacing the System Startup
 
-In most cases customizing the default boot is the better approach, which is documented below. If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card. If this file is present nothing in the system will be auto-started.
+In most cases customizing the default boot is the better approach, which is documented below.
+If the complete boot should be replaced, create a file `/fs/microsd/etc/rc.txt`, which is located in the `etc` folder on the microSD card.
```

Comment:
Should we say `/etc/rc.txt` on the SD card. The `/fs/microsd/` is just how PX4 sees it - and irrelevant to the user at this point.

Reply:
Yes